### PR TITLE
Add MdTooltip-component

### PR DESCRIPTION
**Pull request by aurorascharff**
# Describe your changes

New component MdTooltip. Debounced hover on child element displays information, hides the tooltip for screenreaders and requires an aria-label alternative. Can be closed with ESC.

Animation

## Checklist before requesting a review

- [x] I have performed a self-review and test of my code
- [x] If new component: Is story for component created in `stories`-folder?
- [x] If new component: Is tsx-file import added to `packages/react/index.tsx`?
- [x] If new component: Is css-file added to `packages/css/index.css`?

### CODE REVIEW

**Comment by aurorascharff** [comment]
> Er denne fortsatt aktuell, eller kan den bare lukkes?

Kanskje. Har funnet ut av at jeg burde vente på nye features i Chrome før jeg lager denne funsjonaliteten

File: packages/css/src/tooltip/tooltip.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  border-radius: 4px;
+  padding: 0.5em 1em 0.5em 1em;
+}
+
+.md-tooltip--show {
+  visibility: visible;
+}
+
+.md-tooltip--bottom {
+  transform: translate(-35%, 0.8em);
```
Comment (thomaslarsson) [inline packages/css/src/tooltip/tooltip.css:23]: Same with the different translate values. Should probably be pixels for consistency

**Review by thomaslarsson — APPROVED** [review]
Approved with some minor suggestions.

File: packages/css/src/tooltip/tooltip.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  z-index: 2;
+  background-color: white;
+  visibility: hidden;
+  box-shadow: 0px 4px 4px 0px #00000040;
+  border: 1px solid var(--mdPrimaryColor);
+  font-family: 'Open Sans';
+  height: fit-content;
+  width: fit-content;
+  position: absolute;
+  z-index: 99;
```
[the remark this answers is not in the document]
Comment (aurorascharff) [reply packages/css/src/tooltip/tooltip.css:11]: Oops, tusen takk!

File: packages/css/src/tooltip/tooltip.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  font-weight: 400;
+  font-size: 14px;
+  border-radius: 4px;
+  padding: 0.5em 1em 0.5em 1em;
```
[the remark this answers is not in the document]
Comment (aurorascharff) [reply packages/css/src/tooltip/tooltip.css:15]: Good point. Fix inc.

**Pull request merged**